napari/layers/points/_tests/test_points.py

```diff
@@ -15,6 +15,44 @@ def test_empty_points():
     assert pts.data.shape == (0, 2)
 
 
+def test_empty_points_with_properties():
+    """ Test instantiating an empty Points layer with properties
+        See: https://github.com/napari/napari/pull/1069
+    """
```

Comment:
Minor point: please make sure all docstrings conform to [PEP257](https://www.python.org/dev/peps/pep-0257/). In this case:
```suggestion
    """Test instantiating an empty Points layer with properties.

    See: https://github.com/napari/napari/pull/1069
    """
```